services/repository-managers/src/main/java/org/sagebionetworks/repo/manager/table/TableQueryManagerImpl.java

```diff
@@ -202,7 +202,7 @@ QueryResultBundle queryAsStream(final ProgressCallback progressCallback, final U
 					final TableStatus status = validateTableIsAvailable(query.getTableId());
 					// run the query
 					QueryResultBundle bundle = queryAsStreamAfterAuthorization(progressCallback, query,
-							rowHandler, options);
+							rowHandler, options, user.getId());

```

Comment:
This wont' be needed after the FacetModel is modified to user the user in the original query